.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Handle line endings automatically for files detected as text
```

Comment:
could you explain this change a bit? i'm not fully familiar with gitattributes. also perhaps it could be a separate PR

Reply:
Shoot sorry, I didn’t mean to commit that. Since I’m working in Windows I needed to make sure my line endings were consistent with the rest of the repo. I’ll take it out and update this PR later.

Reply:
Should be good now.